test/integration/smoke-testing/test_statistics_page.js

```diff
@@ -0,0 +1,125 @@
+        .then( function () {
+            driver.findElement(seleniumWebdriver.By.css('g.nv-series'))
+                .then( function (element) {
+                    return element.getText('text');})
+                .then( function (text) {
+                    t.equal(text, chartLegendLabel);})
+                .then( function () {
+                    driver.findElement(seleniumWebdriver.By.xpath('//div[@id="activity_chart"]/*[name()="svg"]' +
+                    '/*[name()="g"]/*[name()="g"]/*[name()="g"]/*[name()="g"]/*[name()="g"]' +
+                    '/*[name()="g"]'))
+                        .then( function (element) {
+                            return element.getText('text');})
+                        .then( function (text) {
+                            t.equal(text, chartXLabel);
+                            t.end();
+                        });
+                });
+        });
+});
+
+// to verify the Active Users Chart's presence, checks the title,
+// a label on the X axis, and a label in the legend
+tap.test('activeUsersChartShouldExist', function (t) {
+    var chartTitle = 'Monthly Active Users';
+    var chartLegendLabel = 'Project Creators';
+    var chartXLabel = '01/2008';
+    driver.findElements(seleniumWebdriver.By
+        .css('div.box-head h3'))
+        .then( function (elements) {
+            var element = elements[1];
+            return element.getText('h3');})
+        .then( function (text) {
+            t.equal(text, chartTitle);})
+        .then( function () {
+            driver.findElements(seleniumWebdriver.By.css('g.nv-series'))
+                .then( function (elements) {
+                    // the 3 labels in the first chart's legend are elements 0 - 2 in this list
+                    var element = elements[3];
+                    return element.getText('text');})
+                .then( function (text) {
+                    t.equal(text, chartLegendLabel);})
+                .then( function () {
+                    driver.findElement(seleniumWebdriver.By.xpath('//div[@id="active_user_chart"]/*[name()="svg"]' +
+                    '/*[name()="g"]/*[name()="g"]/*[name()="g"]/*[name()="g"]/*[name()="g"]' +
+                    '/*[name()="g"]'))
+                        .then( function (element) {
+                            return element.getText('text');})
+                        .then( function (text) {
+                            t.equal(text, chartXLabel);
+                            t.end();
+                        });
+                });
+        });
+});
+
+// to verify the Comment Activity Chart's presence, checks the title,
+// a label on the X axis, and a label in the legend
+tap.test('commentActivityChartShouldExist', function (t) {
+    var chartTitle = 'Monthly Comment Activity';
+    var chartLegendLabel = 'Stacked';
+    var chartXLabel = '01/2008';
+    driver.findElements(seleniumWebdriver.By
+        .css('div.box-head h3'))
+        .then( function (elements) {
```

Comment:
Actually sorry just noticed this – I think the style we generally use in www is to newline the closing brace/parentheses, like:
```javascript
.then(function (text) {
    t.equal(text, chartTitle);
})
.then...
```
rather than closing on the same line